src/utilities/stan-interface.jl

```diff
@@ -66,6 +66,7 @@ function sample(
     end
 end
 
+# FIXME: where is `spl` in the code below?
```

Comment:
Seriously where is `spl` in the code below? 😆

Reply:
Is this function even used?

Reply:
Might not be correctly tested since whatever.